lib/spotlight/dor/indexer.rb

```diff
@@ -1,17 +1,20 @@
+# rubocop:disable Metrics/ClassLength
 # external gems
 require 'gdor/indexer'
 require 'solrizer'
-# Base class to harvest from DOR via harvestdor gem
+
 module Spotlight::Dor
+  # Base class to harvest from DOR via harvestdor gem
   class Indexer < GDor::Indexer
     # add contentMetadata fields
+    # rubocop:disable Metrics/LineLength
     before_index do |sdb, solr_doc|
-      Solrizer.insert_field(solr_doc, 'content_metadata_type', sdb.public_xml.xpath("/publicObject/contentMetadata/@type").text, :symbol, :displayable)
+      Solrizer.insert_field(solr_doc, 'content_metadata_type', sdb.public_xml.xpath('/publicObject/contentMetadata/@type').text, :symbol, :displayable)
```

Comment:
I guess Hound doesn't care about the inline disabling (and, thinking about it, if it's only looking at the diffs, I don't see how it could do otherwise).